doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1661,6 +1661,161 @@ Testing boolean expressions: guard
    .. exn:: Condition not satisfied.
       :undocumented:
 
+Checking properties of terms
+----------------------------
+
+Each of the following tactics acts as the identity if the check
+succeeds, and results in an error otherwise.
+
+.. tacn:: constr_eq @one_term @one_term
+
+   Succeeds if the arguments are equal ignoring differences in alpha
+   conversion, casts and universe constraints and fails otherwise.
```

Comment:
It is the sound of one hand clapping :-)

Reply:
This one checks if the arguments are equal up to adding some universe equations.
_strict checks if they are equal with the current universe equations.
_nounivs just ignores universes.

So I would say something like
~~~
constr_eq_strict

Succeeds if the arguments are equal modulo alpha conversion and ignoring casts. Universes are considered equal when they are equal in the universe graph.

constr_eq

Like constr_eq_strict, but may add constraints to make universes equal.

constr_eq_nounivs

Like constr_eq_strict, but all universes are equated.
~~~

Reply:
also IIRC _nounivs is the one used by ltac `match`, eg 
~~~coq
  let c := constr:(Type) in
  let x := constr:(Type : c) in
  match c with
  | x => idtac c x
  end.
~~~
succeeds.

Reply:
@SkySkimmer I like your suggested text, though I'd suggest "but all universes are considered equal." instead of "but all universes are equated." for `constr_eq_nounivs`